hype-submitter/src/main/java/com/spotify/hype/runner/LocalDockerRunner.java

```diff
@@ -68,18 +68,9 @@ public LocalDockerRunner(final DockerClient client,
   }
 
   private String getImageWithTag(final RunSpec runSpec) {
-    final RunEnvironment.EnvironmentBase base = runSpec.runEnvironment().base();
-    if (base instanceof RunEnvironment.SimpleBase) {
-      final RunEnvironment.SimpleBase simpleBase = (RunEnvironment.SimpleBase) base;
-      return simpleBase.image().contains(":")
-          ? simpleBase.image()
-          : simpleBase.image() + ":latest";
-    } else if (base instanceof RunEnvironment.YamlBase) {
-      throw
-          new UnsupportedOperationException(
-              "Yaml based environment not supported in local mode yet!");
-    }
-    throw new IllegalArgumentException("Illegal type of run environment " + base.toString());
+    return runSpec.image().contains(":")
+           ? runSpec.image()
+           : runSpec.image() + ":latest";
```

Comment:
The `:latest` addition is done here but not in `KubernetesDockerRunner`. I think we can just omit this behaviour and let docker/k8s deal with the image name vanilla.